specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/1.0.0/iotcentral.json

```diff
@@ -0,0 +1,2262 @@
+            "description": "Individual device attestation body.",
+            "schema": {
+              "$ref": "#/definitions/Attestation"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Attestation"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "Devices_UpdateAttestation",
+        "summary": "Update an individual device attestation via patch",
+        "x-ms-examples": {
+          "Update device attestation - Symmetric Key": {
+            "$ref": "./examples/devices_attestations_set_symmetricKey.json"
+          },
+          "Update device attestation - TPM": {
+            "$ref": "./examples/devices_attestations_set_tpm.json"
+          },
+          "Update device attestation - X509": {
+            "$ref": "./examples/devices_attestations_set_x509.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Individual device attestation patch body.",
+            "schema": {
+              "type": "object"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Attestation"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "Devices_RemoveAttestation",
+        "summary": "Remove an individual device attestation",
+        "x-ms-examples": {
+          "Delete device attestation": {
+            "$ref": "./examples/devices_attestations_delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          }
+        }
+      }
+    },
+    "/devices/{device_id}/commands/{command_name}": {
+      "get": {
+        "operationId": "Devices_GetCommandHistory",
+        "summary": "Get device command history",
+        "x-ms-examples": {
+          "Get command history": {
+            "$ref": "./examples/devices_commandshistory_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "$ref": "#/parameters/CommandName"
+          }
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceCommandCollection"
+            }
+          }
+        }
+      },
+      "post": {
+        "operationId": "Devices_ExecuteCommand",
```

Comment:
IIRC there is PolyCheck guidance against using the word "execute". We might prefer to use "run" or something else instead.

Reply:
Will do the update, would you please point me the link of IIRC ploy check guidance? Thanks!

Reply:
I'll send it to you in Teams.

Reply:
Fixed.